ivy/functional/backends/numpy/experimental/linear_algebra.py

```diff
@@ -97,7 +97,7 @@ def kron(
 
 
 @with_supported_dtypes(
-    {"1.11.0 and below": ("float32", "float64", "complex64", "complex128")},
+    {"2.0.1 and below": ("float32", "float64", "complex64", "complex128")},
```

Comment:
isn't this meant to be a numpy latest version instead of torch (2.0.1)

Reply:
yep